src/sources/aws_sqs/source.rs

```diff
@@ -83,7 +83,7 @@ impl SqsSource {
         let receive_message_output = match result {
             Ok(output) => output,
             Err(err) => {
-                error!("SQS receive message error: {:?}", err);
+                error!("SQS receive message error: {:?}.", err);
```

Comment:
We should ideally not be using format strings in structured output, though `check-events` isn't scanning for this yet. We can tackle that in a separate issue.

Reply:
Should this also be an internal event?